app/js/preloads/downloads.js

```diff
@@ -137,11 +140,13 @@ const addDownload = (download, id) => {
     downloadUrl.href = download.url
     downloadUrl.textContent = decodeURIComponent(download.url)
     misc.appendChild(downloadUrl)
-    const file = document.createElement("span")
+    const file = document.createElement("a")
```

Comment:
I don't like misusing elements for what they are not made for, which in this case is done by prevent the default click action of a link and sending the open ipc. I think it's nice to either have the entire block clickable with a simple title explaining the action, or keep it as is (possibly with just a hover title on the filename).

Reply:
The only difference to a link is that the url doesn't open in vieb, but in an external program.  Before, I found it confusing that it wasn't styled as a link (blue and underlined).  I can change it back to a span, but I'd really like to keep the link styling (or some other indication that the file name can be clicked).

Reply:
I get what you're saying, and I agree that we can improve the click functionality. Though I think that making the title clickable along with adding the hover text is good enough.  If people expect to be able to open the file with a click, the title and pointer cursor should be all the feedback they need to do so, and if using follow mode they will see the onclick outline as well.

As mentioned I don't like it being a link if it doesn't behave as one, interval Vieb pages should work as predictable as possible combined with follow mode, and this doesn't do that, let aside the fact that it's semantically [discouraged](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#onclick_events) to use a link for this.

Reply:
> Though I think that making the title clickable along with adding the hover text is good enough.

This line of the diff is about the filename three lines below the title.  Are you suggesting that we should remove the filename and only have the title be clickable?  I kind of like this to be honest, the download div is a bit overcrowded.  (In any case, I'll add a tooltip to the title as well.)

Reply:
I meant that both the filelocation and the title can have the hover text and be clickable, but that changing the filelocation to a link shouldn't be necessary for that.

Reply:
Although it is crowded, I don't like removing it, as it contains info that isn't presented elsewhere in the download: the save location (which can be very handy in case of duplicate filenames, or when having recently changed the downloadpath).

Reply:
Ok, I've reverted the change and added the hover text.

Reply:
Great, but there is one other thing we need to do that I just realized. The reason I used the textContent in the click event handler is because the download element gets updated when new files are added, or previous ones are removed. By using `download.file` instead, there will be disconnect between the click action and the filename displayed. The same is actually the case for the title. We can either use the textContent for both the handlers, or change the update function to remove the old listener and add the correct one.

Reply:
Oh, I've changed it to use `file.textContent` in both cases.  I'm not going to add code to update event handlers.  It's probably a better idea to replace the whole element if a download has changed (or just use react to minimize changes to the dom).